2.7/Dockerfile

```diff
@@ -20,12 +20,13 @@ building and running various Ruby $RUBY_VERSION applications and frameworks. \
 Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
 It has many features to process text files and to do system management tasks (as in Perl). \
 It is simple, straight-forward, and extensible." \
-    PATH=$PATH:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin \
-    LD_LIBRARY_PATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/lib64:/opt/rh/$NODEJS_SCL/root/usr/lib64 \
-    X_SCLS=rh-ruby$RUBY_SCL_NAME_VERSION \
-    MANPATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/$NODEJS_SCL/root/usr/share/man \
-    PYTHONPATH=/opt/rh/$NODEJS_SCL/root/usr/lib/python2.7/site-packages
-
+    PATH=$PATH:/opt/rh/$NODEJS_SCL/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin \
+    LD_LIBRARY_PATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/lib64:/opt/rh/$NODEJS_SCL/root/usr/lib64:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/lib64 \
+    X_SCLS=$X_SCLS:rh-ruby$RUBY_SCL_NAME_VERSION \
+    MANPATH=/opt/rh/$NODEJS_SCL/root/usr/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/man:/share/man/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/$NODEJS_SCL/root/usr/share/man \
+    PYTHONPATH=/opt/rh/$NODEJS_SCL/root/usr/lib/python2.7/site-packagesi \
```

Comment:
This looks wrong (the last letter).

Also I'm not sure tis path makes sense- nodejs + python?

Reply:
Thanks for the typo it could be a good point for tests improvement, I think, nodejs is present in all containers and why shouldn't we enable it?

Reply:
It was not about enabling `nodejs`, I'm fine with that, I simply don't understand the `PYTHONPATH` path:

```
     PYTHONPATH=/opt/rh/$NODEJS_SCL/root/usr/lib/python2.7/site-packagesi \ 
```
Is that needed for nodejs?

Reply:
@pvalena I agree it's not necessary, but tests require all variables which are enabled after the container starts. For removing this path is it probably necessary to update scl_enable script? But from my point of view exported PYTHONPATH isn't a problem, because nowadays it's still present in container env.

Reply:
@fila43 I'm not saying it's not necessary. What I was trying to say is that I have no Idea how that works, and what it's for.

If it's used for tests, and is a standard container ENV, then it's fine with me.

Reply:
yes, it's some nodejs magic, are we able to consider that this PR is legitimate?

Reply:
I said I'm fine with it.
Please don't interpret my noting my (below-the-line in italics) note about __my__ lack of knowledge as a disagreement. Especially when it states otherwise.